test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -1482,9 +1482,8 @@ BOOST_AUTO_TEST_CASE(now)
 			}
 		}
 	)";
-	m_rpc.test_modifyTimestamp(0x776347e2);
```

Comment:
Please add an additional assert that it changed between these calls, just to double check that the backend actually works.